packages/venia-concept/tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+const venia = {
+    mode: 'jit',
+    purge: {
+        content: ['./src/**/*.css'],
+        extractors: [
+            {
+                extractor: content => content.match(matcher) || [],
+                extensions: ['css']
+            }
+        ]
+    },
+    separator: '_'
+};
+
+const config = {
+    presets: [venia]
```

Comment:
@larsroettig Nice work. I think we should list `mode`, `separator`, and `purge` here in the base `tailwind.config.js` file. These properties generally shouldn't change; if they do, the developer should change them here rather than relying on the last preset to set them correctly.

```suggestion
    mode: 'jit',
    purge: { /* ... */ },
    separator: '_',
    // if you created a theme, apply it here
    presets: [venia]
```